### PR TITLE
feat: Image Hotspot Creator to the market place

### PR DESCRIPTION
**Pull request by vijayaragavan462**
## Purpose

We are introducing our Image Hostpot Creator application to the Contentful Market place.

### CODE REVIEW

**Comment by surya-sudoboat** [comment]
@gavinmatt Could you please clarify for us what the precise problem with apps-ci-pipeline is?

**Comment by matthew-gordon** [comment]
> @gavinmatt Could you please clarify for us what the precise problem with apps-ci-pipeline is?

Hi @surya-sudoboat! I am one of the maintainers of this repository and I'll work to try and unblock these build failures with you.

I notice in the submission there has been an additional `config.yaml` for the build pipeline added under the `.contentful/.circleci` directory, full path being `.contentful/.circleci/config.yaml`. 

This may be causing conflicts within the platform, and the first suggestion is to completely remove the entire `.circleci` directory from under `.contentful`.

File: .contentful/.circleci/config.yml
Comment (matthew-gordon) [inline .contentful/.circleci/config.yml:1]: This file and directory `.circleci` should be removed from the project, we maintain a root level `.circleci/config.yaml`

**Review by matthew-gordon — CHANGES_REQUESTED** [review]
Not entirely sure, everything looks good except for the duplicate `.circleci/config.yaml` directory and config file. Try deleting these from the `.contentful` directory and we can re-run the app pipeline 👍

**Comment by surya-sudoboat** [comment]
> Not entirely sure, everything looks good except for the duplicate `.circleci/config.yaml` directory and config file. Try deleting these from the `.contentful` directory and we can re-run the app pipeline 👍

Thanks for your help @matthew-gordon. We will remove the respective files and raise the pr again

**Pull request closed without merging**
